cypher-shell/src/main/java/org/neo4j/shell/prettyprint/PrettyPrinter.java

```diff
@@ -121,14 +118,14 @@ private String mapAsString(Map<String, Object> map) {
         StringBuilder sb = new StringBuilder("{");
         sb.append(
                 map.entrySet().stream()
-                        .map(e -> escapeIfNeeded(e.getKey()) + COLON_SEPARATOR + e.getValue())
+                        .map(e -> cypherVariablesFormatter.escape(e.getKey()) + COLON_SEPARATOR + e.getValue())
```

Comment:
Would really like it if the method were static and you had a static import for it this became:

```
.map(e -> escape(e.getKey()) ...
```